lib/galaxy/tool_util/client/staging.py

```diff
@@ -261,11 +265,11 @@ def use_fetch_api(self):
 
 
 class InteractorStaging(StagingInterface):
-    def __init__(self, galaxy_interactor, use_fetch_api=DEFAULT_USE_FETCH_API):
+    def __init__(self, galaxy_interactor: "ApiTestInteractor", use_fetch_api: bool = DEFAULT_USE_FETCH_API) -> None:
         self.galaxy_interactor = galaxy_interactor
         self._use_fetch_api = use_fetch_api
 
-    def _post(self, api_path, payload, files_attached=False):
+    def _post(self, api_path: str, payload: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
This is used by Planemo - is it worth breaking these interfaces?

Reply:
I'm removing this also in `PlanemoStagingInterface._post()`, if that's OK.

Reply:
@jmchilton On second thought, I think this is fine since in Planemo this is subclassed in `PlanemoStagingInterface` but having an extra argument in the method is fine.
On the other hand, we cannot (yet) remove `files_attached` from `PlanemoStagingInterface._post()` because then it would break when using pre-23.01 galaxy-tool-util.